scylladb/scylladb_test.go

```diff
@@ -0,0 +1,151 @@
+package scylladb
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+var testStore = New(Config{Reset: true})
+
+func Test_Scylla_Set(t *testing.T) {
+	// Create a new instance of the Storage
+	var (
+		key   = "john"
+		value = []byte("doe")
+	)
+	err := testStore.Set(key, value, time.Minute)
+	require.NoError(t, err)
+}
+
+func Test_Scylla_Set_Override(t *testing.T) {
+	var (
+		key = "john"
+		val = []byte("doe")
+	)
+
+	err := testStore.Set(key, val, 0)
+	require.NoError(t, err)
+
+	err = testStore.Set(key, val, 0)
+	require.NoError(t, err)
+}
+
+func Test_Scylla_Get(t *testing.T) {
+	var (
+		key = "john"
+		val = []byte("doe")
+	)
+
+	err := testStore.Set(key, val, 0)
+	require.NoError(t, err)
+
+	result, err := testStore.Get(key)
+	require.NoError(t, err)
+	require.Equal(t, val, result)
+}
+
+func Test_Scylla_Set_Expiration(t *testing.T) {
+	var (
+		key = "john"
+		val = []byte("doe")
+		exp = 1 * time.Second
+	)
+
+	err := testStore.Set(key, val, exp)
+	require.NoError(t, err)
+
+	time.Sleep(1100 * time.Millisecond)
```

Comment:
There needs to be a `get()` after the sleep, to verify that the `key` expired after 1sec